Tests/Encryptor/EncryptorTest.php

```diff
@@ -35,4 +45,43 @@ public function testEncryptAndDecrypt(): void
 
         $this->assertSame($rawData, $encryptor->decrypt($encryptedData));
     }
+
+    /**
+     * @expectedException        \Ekino\DataProtectionBundle\Exception\EncryptionException
+     * @expectedExceptionMessage Unexpected failure in openssl_encrypt.
```

Comment:
don't use annotation for expectedException plz as it's deprecated with phpunit8 and will be removed on phpunit9